def raiseException(age):
    if age<0:
        raise ValueError("This error raised by myself")
